### ISSUE TITLE
More Component Issues

### ISSUE THREAD

**Issue by devondragon** [devondragon/SpringCFTurnstile#3]
There are further issues that are only being caught when using the library in a real application. Part of the issue is the current test classes are under the same package as the library code, leading to the component scanning picking up library components, without the library having to vend them correctly.

I need to move the test classes, and fix the latest issue.

**Comment by devondragon** [comment]
Hopefully fixed.  Pushing new build to Maven central and will test from there.

**Comment by devondragon** [comment]
Some very odd issue with the application.yml (or application.propertiers), not getting read by a consuming application. Refactored this project to better mirror my Spring AI Client library project which has no issues with the configuration being loaded by the consuming application, but had no luck.

Found several likely related posts:

https://stackoverflow.com/questions/34056634/project-as-application-yml-was-not-loaded-in-project-b-that-has-a-dependency-on
https://stackoverflow.com/questions/60892465/spring-boot-cannot-read-application-properties-from-a-dependency-jar
https://stackoverflow.com/questions/65625859/not-clear-why-my-second-application-yml-is-not-loaded

So it seems that having the same named config files doesn't work right.  

I've moved and renamed the file and am referring to it directly from my properties configuration class.  Seems to be working now!
